package.json

```diff
@@ -22,7 +22,9 @@
     "start:test": "yarn-or-npm run run-bullmq && ENV=test node src/listener.js",
     "start:local": "docker-compose -f docker-compose-local.yml up --force-recreate --build -d",
     "logs": "docker logs -f --tail 1000 warp-dre-node-dre-1",
-    "stop:local": "docker logs -f --tail 1000 warp-dre-node-dre-1"
+    "stop:local": "docker logs -f --tail 1000 warp-dre-node-dre-1",
+    "migrate:make:state": "npx knex --knexfile=./knexConfigStateDb.js migrate:make",
```

Comment:
this has to be run within a container, right?

Reply:
no, it's just a script to create a migration template. the script to run the latest migration is executed inside `listener.js` file but i assume we should run it within container.